src/eureka/S4_generate_lightcurves/s4_genLC.py

```diff
@@ -223,44 +226,61 @@ def genlc(eventlabel, ecf_path=None, s3_meta=None):
                              f'wavelength',
                              mute=meta.verbose)
 
-            # Apply 1D drift/jitter correction
-            if meta.correctDrift:
+            if meta.record_ypos:
```

Comment:
For now, to avoid breaking backwards compatibility I would write:
```
if hasattr(meta, 'record_ypos') and meta.record_ypos:
```

Reply:
done